src/Crypto/pbc-cffi.lisp

```diff
@@ -555,7 +555,9 @@ comparison.")
 ;; -------------------------------------------------
 
 (defun need-pairing ()
+  (format *standard-output* "~%trying init-pairing~%")
   (unless *curve*
+    (format *standard-output* "~%running init-pairing~%")
     (init-pairing)))
 
```

Comment:
Do you want it to let you know here that it is done with the try, after the (unless ...)?